Makefile

```diff
@@ -573,9 +579,12 @@ new-service: .venv ## Bakes a new project from cookiecutter-simcore-pyservice an
 
 
 .PHONY: openapi-specs
-openapi-specs: ## bundles and validates openapi specifications and schemas of ALL service's API
-	@$(MAKE_C) services/web/server $@
-	@$(MAKE_C) services/storage $@
+openapi-specs: .env _check_venv_active ## generates and validates openapi specifications and schemas of ALL service's API
+	@for makefile in $(MAKEFILES_WITH_OPENAPI_SPECS); do \
+		echo "Generating openapi-specs using $${makefile}"; \
+		$(MAKE_C) $$(dirname $${makefile}) install-dev && $(MAKE_C) $$(dirname $${makefile}) $@; \
```

Comment:
MINOR: I think multiline of 585 is more readable. I would like to see something that reminds my
```
make install-dev
make openapi-specs
```

Reply:
done, thanks